src/components/Property/Contacts/TenantsContactTable/TenantPhoneNumber.js

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react'
+import RemoveContactModal from './modals/RemoveContactModal'
+import SetAsMainModal from './modals/SetAsMainModal'
+
+const TenantPhoneNumber = ({ phoneNumber, tenant, reloadContacts }) => {
+  const [showRemoveModal, setShowRemoveModal] = useState(false)
+  const [showMainContactModal, setShowMainContactModal] = useState(false)
+
+  const isMainNumber = phoneNumber.subType === 'mainNumber'
+
+  return (
+    <li className="tenantContactsTable-phoneNumberGrid govuk-body">
+      <div className="tenantContactsTable-valueGroup">
+        <h4 className="tenantContactsTable-phoneSubType">
+          {phoneNumber.subType}
+        </h4>
+        <div className="tenantContactsTable-phone">{phoneNumber.value}</div>
+      </div>
+      {!!phoneNumber?.description && (
+        <div className="tenantContactsTable-description">
+          {phoneNumber.description}
+        </div>
+      )}
+      <div className="tenantContactsTable-buttonGroup">
+        <button
+          type="button"
+          className="tenantContactsTable-button"
+          onClick={() => setShowMainContactModal(true)}
+          disabled={isMainNumber}
+          style={
+            isMainNumber
```

Comment:
Could this be simplified to `{isMainNumber && {color: '#888'}}`?

Reply:
I think using a class is a much better approach.

![image](https://github.com/LBHackney-IT/repairs-hub-frontend/assets/88662046/71b818a1-38df-4ff2-b2e8-94eaa60f1885)

Reply:
I saw your commit and it like a good solution, agreed!